src/client/components/app/Navbar/index.tsx

```diff
@@ -25,6 +25,10 @@ const StyledNavbarActions = styled.div`
   }
 `;
 
+const StyledLogo = styled.div`
+  margin-right: 1rem;
+`;
```

Comment:
can you also verify rest of the margins are following designs here?

Reply:
Created an overlay, it appeared that it had indeed a little bit more margin right, fixed with https://github.com/yearn/yearn-finance-v3/pull/722/commits/ec931b80a2d92d066417691abd256ef40839c3e3

<img width="303" alt="Screen Shot 2022-06-29 at 2 40 18 pm" src="https://user-images.githubusercontent.com/78794805/176428264-baf19d95-452d-4814-8133-0e7b1c6ab107.png">